src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md

```diff
@@ -37,11 +37,7 @@ During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud
 ### Editing and scaling
```

Comment:
If you still can't scale storage we should say that on this page somewhere

Reply:
I'm waiting on @adwittumuluri to confirm.

Reply:
Confirmed that you _will_ be able to scale storage with the release targeted for 9/22, and by extension the GA release.

Reply:
And you can only scale it UP, not back DOWN, yes?

Reply:
that is correct, [docs](https://learn.microsoft.com/en-us/troubleshoot/azure/virtual-machines/downsize-data-disk-without-losing-data?tabs=windows)

> Downsizing a disk directly isn't supported because of the risk of data loss.